data/mods/Magiclysm/enchanted/enchanted_belts.json

```diff
@@ -0,0 +1,186 @@
+    "coverage": 10,
+    "encumbrance": 3,
+    "weight_capacity_bonus": "10 kg",
+    "use_action": {
+      "type": "holster",
+      "holster_prompt": "Stick what into your belt",
+      "holster_msg": "You tuck your %s into your %s",
+      "min_volume": "1 ml",
+      "max_volume": "4 L",
+      "max_weight": "5 kg",
+      "draw_cost": 5,
+      "multi": 4,
+      "skills": [
+        "pistol",
+        "shotgun",
+        "cutting",
+        "archery",
+        "bashing",
+        "launcher",
+        "melee",
+        "rifle",
+        "smg",
+        "stabbing",
+        "throw",
+        "ALL",
+        "gun",
+        "unarmed"
+      ],
+      "flags": [ "SHEATH_KNIFE", "SHEATH_SWORD", "BELT_CLIP", "SHEATH_SPEAR" ]
+    }
+  },
+  {
+    "type": "ARMOR",
+    "id": "mbelt_technomancer_toolbelt",
+    "name": "technomancer's toolbelt",
+    "name_plural": "technomancer's toolbelts",
+    "weight": "2000 g",
+    "color": "brown",
+    "covers": [ "TORSO" ],
+    "storage": "1 L",
+    "symbol": "[",
+    "description": "A custom-built leather utility belt covered with straps and pouches containing many useful hand tools and a sheath to carry a smaller blade.  Durable and carefully crafted to be comfortable to wear.  Activate to sheathe/draw a weapon.",
+    "price": 500000,
+    "material": [ "leather", "steel" ],
+    "volume": "4 L",
+    "flags": [ "VARSIZE", "WATER_FRIENDLY", "STURDY", "WAIST", "OVERSIZE" ],
+    "coverage": 10,
+    "encumbrance": 4,
+    "material_thickness": 4,
+    "use_action": [
+      {
+        "type": "holster",
+        "holster_prompt": "Sheath blade",
+        "holster_msg": "You sheath your %s",
+        "draw_cost": 3,
+        "min_volume": "250 ml",
+        "max_volume": "1 L",
+        "flags": [ "SHEATH_KNIFE", "SHEATH_SWORD" ]
+      },
+      "CROWBAR"
+    ],
+    "qualities": [
+      [ "HAMMER", 3 ],
+      [ "PRY", 1 ],
+      [ "SAW_M", 2 ],
+      [ "SAW_W", 2 ],
+      [ "WRENCH", 2 ],
+      [ "AXE", 1 ],
+      [ "SCREW", 1 ],
+      [ "HAMMER_FINE", 1 ],
+      [ "SAW_M_FINE", 1 ],
+      [ "WRENCH_FINE", 1 ],
+      [ "SCREW_FINE", 1 ]
+    ]
+  },
+  {
+    "type": "TOOL_ARMOR",
+    "copy-from": "mbelt_leather",
+    "id": "mbelt_iron_whip",
+    "name": "Belt of The Iron Whip",
+    "name_plural": "Belts of the Iron Whip",
+    "volume": "1 L",
+    "material": [ "steel" ],
+    "description": "A braided but flexible belt seemingly made of metal.  You can activate it to transform it into a whip and flay your enemies.",
+    "material_thickness": 4,
+    "flags": [ "STURDY" ],
+    "use_action": {
+      "type": "transform",
+      "target": "mwhip_iron",
+      "active": true,
+      "moves": 20,
+      "msg": "You grab the belt and it uncoils to become a flexible metal whip in your hand!"
+    }
+  },
+  {
+    "id": "mwhip_iron",
+    "type": "GENERIC",
+    "category": "weapons",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "The Iron Whip",
```

Comment:
Is this supposed to be uppercase?

Reply:
That is intended, yes. It's basically a named item I guess you would say?